docs/tasks/announcements.rst

```diff
@@ -1,46 +1,87 @@
+=============
 Announcements
 =============
 
-You mean there's more than place for announcements?
----------------------------------------------------
+As of 2018, there is only one place to write announcements: `CampusGroups <campusgroups.rit.edu>`__.
+Announcements are generally sent to RITlug club members at the beginning of each week.
+These weekly emails inform club members about the topic of the upcoming general meeting.
+
+
+How to Create an Announcement
+=============================
+
+In order to post a club-wide announcement, you need to navigate to the RITlug `CampusGroups <https://campusgroups.rit.edu>`__ page.
+On the main club page, go to the *Emails* drop-down, and select the orange *Compose Email* button.
+To send an email from here, one must:
+
+#. Add and review list of recipients
+#. Choose sender email address
+#. Select email template
+#. Compose email
+#. Send email
+
+Selecting Recipients
+--------------------
```

Comment:
Please use the mailing list that's already been set up with the correct groups. Email > Mailing List and then hit send on the list there

Reply:
I shall look into that :+1:

Reply:
@Tjzabel No, we have past members who like to like to get updates (and they can turn it off if they don't), and "contacts" is needed for the mailing list forwarding